src/explore-education-statistics-common/src/styles/utils/_overflow.scss

```diff
@@ -1,3 +1,7 @@
 .dfe-overflow-x--auto {
   overflow-x: auto;
 }
+
+.dfe-overflow-y--scroll {
+  overflow-y: scroll;
+}
```

Comment:
Seems like there's a bit of a mistake here though - it should be `dfe-overflow-y--auto` to avoid the scroll bar showing if the max height hasn't been exceeded.

It needs the `overflow-y: auto` as the style as well.

Reply:
Oh yeah, changed to: 

```
.dfe-overflow-y--auto {
  overflow-y: auto;
}
```

Assuming you didn't mean the following (due to the last `overflow-y` rule taking precedence): 

```
.dfe-overflow-y--auto {
  overflow-y: auto;
  overflow-y: scroll;
}
```

Reply:
Yep, the first one is what I meant :wink: